### PR TITLE
Explain why MPTCP is not sufficient

### PR DESCRIPTION
**Pull request by boucadair**
More text is needed, but this is a start

### CODE REVIEW

**Review by SpencerDawkins — COMMENTED** [review]
ISTM that the second paragraph of this text goes a bit further into detail than we need to, at this point. I think it's fair to say (as the first paragraph says) that MPTCP isn't expecting to distribute UDP flows (as per Rel-17), and to add (I'm assuming) MPTCP isn't expecting to distribute IP flows (as per Rel17). I'm not sure we need to say more there. 

The point about 3GPP having ATSSS modes that "the base QUIC specification" doesn't support is useful if we're expecting IETF to add behaviors that would support those modes. Is that part of the request to IETF? 

(I should also ask which "base QUIC specification" you're looking at - I just want to understand this better, and thanks for bringing it up). 

What are the other problems with MPTCP that we would hope the IETF would want to solve in MP-QUIC?

**Comment by boucadair** [comment]
> (I should also ask which "base QUIC specification" you're looking at - I just want to understand this better, and thanks for bringing it up).

I'm referring to draft-ietf-quic-transport.

File: draft-quic-atsss-reqs.mkd [hunk 1]
```diff
@@ -125,6 +125,17 @@ The two scenarios that we consider in this document are:
...
+
+Some proposals were made in the past to encapsulate UDP in MPTCP (by means of header swapping and extraction of the UDP payload). For example, early versions of {{I.D-boucadair-mptcp-plain-mode}} specified how UDP flows, in particular, can be distributed among available paths by leveraging MPTCP capabilities. It does so by including a “Protocol” TLV in the MPTCP payload; doing so allows an MPTCP Proxy to unambiguously identify UDP packets vs. native TCP connections. Some modifications are also required at the MPTCP Proxy to avoid HOL blocking issues of UDP-encapsulated data. This design approach is sub-optimal if the UDP-encapsulated data carry its own congestion control signals (that are not visible to the MPTCP proxy because these signals are encrypted such as in QUIC). 
+
+Some ATSSS modes can be natively supported by the base QUIC specification. For example, the “Active-Standby” and “Smallest Delay” steering modes can be supported directly between an UE and a QUIC server without any assistance from the network other than the performance measurement information.
```
Comment (mpiraux) [inline draft-quic-atsss-reqs.mkd:132]: Which are the modes that can't ? Also, consider other WG-adopted drafts as part of the QUIC spec. Not only the -transport draft.

File: draft-quic-atsss-reqs.mkd [hunk 2]
```diff
@@ -125,6 +125,17 @@ The two scenarios that we consider in this document are:
...
+
+Some proposals were made in the past to encapsulate UDP in MPTCP (by means of header swapping and extraction of the UDP payload). For example, early versions of {{I.D-boucadair-mptcp-plain-mode}} specified how UDP flows, in particular, can be distributed among available paths by leveraging MPTCP capabilities. It does so by including a “Protocol” TLV in the MPTCP payload; doing so allows an MPTCP Proxy to unambiguously identify UDP packets vs. native TCP connections. Some modifications are also required at the MPTCP Proxy to avoid HOL blocking issues of UDP-encapsulated data. This design approach is sub-optimal if the UDP-encapsulated data carry its own congestion control signals (that are not visible to the MPTCP proxy because these signals are encrypted such as in QUIC). 
+
+Some ATSSS modes can be natively supported by the base QUIC specification. For example, the “Active-Standby” and “Smallest Delay” steering modes can be supported directly between an UE and a QUIC server without any assistance from the network other than the performance measurement information. 
+
+Nevertheless, blindly distributing packets among existing paths may lead to complications such as:
```
Comment (mpiraux) [inline draft-quic-atsss-reqs.mkd:134]: Are these problems you consider going to be addressed using QUIC for ATSSS ?

File: draft-quic-atsss-reqs.mkd (hunk 1 above)
Comment (boucadair) [reply draft-quic-atsss-reqs.mkd:132]: See the example starting with "blindly distributing packets ...".

**Pull request merged**